src/NavigationActions.js

```diff
@@ -23,6 +25,8 @@ const reset = createAction(RESET);
 const setParams = createAction(SET_PARAMS);
 const uri = createAction(URI);
 
+export const creators = { back, init, navigate, reset, setParams, uri };
```

Comment:
Would this be better to be named as `actionCreators`? Your comment in the `export default` clarifies that these are action creators. Curious on your thoughts just so we remove any ambiguity in the name.